extensions/azurecore/src/extension.ts

```diff
@@ -83,7 +83,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 
 	}
 	updatePiiLoggingLevel();
-
+	let proxyUrl: string | undefined = vscode.workspace.getConfiguration(Constants.HttpConfigSection).get(Constants.Proxy);
```

Comment:
can these let variables be converted to const, I typically make them const unless they need to be modified